enterprise/views.py

```diff
@@ -499,7 +503,23 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_uuid, course
             logger.error('Unable to find course details for course ID: %s', course_id)
             raise Http404
 
-        platform_name = configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)
+        enterprise_customer = get_enterprise_customer_or_404(enterprise_uuid)
+
+        enrollment_client = EnrollmentApiClient()
+        enrolled_courses = enrollment_client.get_enrolled_courses(request.user.username)
```

Comment:
Can we use [this](https://github.com/edx/edx-platform/blob/master/common/djangoapps/enrollment/views.py#L86) enrollment API endpoint instead so that we don't have to retrieve all of the learner's enrollments?